ResizeImage.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import os
+from tempfile import mkstemp
+from urllib import unquote_plus
+
+from PIL import Image
+import boto3
+
+s3 = boto3.client('s3')  # pylint: disable=C0103
+
+log = logging.getLogger(__name__)  # pylint: disable=C0103
+log.setLevel(logging.DEBUG)
+
+
+def handler(event, _):
+    for record in event['Records']:
+        bucket = record['s3']['bucket']['name']
+        # S3 keys are urlencoded in event notifications
+        # https://docs.aws.amazon.com/AmazonS3/latest/dev/notification-content-structure.html
+        key = unquote_plus(record['s3']['object']['key'])
+        size = record['s3']['object']['size']
+        log.debug("Got new file: %s:%s, size %d", bucket, key, size)
+        original = get_file(bucket, key)
+        resized = resize_image(original)
+        put_file(resized,
+                 os.environ.get('DEST_BUCKET', 'calligre-images'),
```

Comment:
Yeah, in global namespace. Just having vars like this all in one place is much easier to find that have to read every line.